### PR TITLE
Updating delete to disconnect targets

### PR DESCRIPTION
**Pull request by oumkale**
Signed-off-by: Oum Kale <oumkale@chaosnative.com>

## Proposed changes

Updating the delete Agent to disconnect Agent
## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

## Special notes for your reviewer:
Screenshot from 2021-03-18 12-11-18

### CODE REVIEW

**Review by ishangupta-ds — DISMISSED** [review]
LGTM

File: litmus-portal/frontend/src/views/Targets/BrowseCluster/TableData.tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ interface TableDataProps {
 const TableData: React.FC<TableDataProps> = ({ data, deleteRow }) => {
   const classes = useStyles();
   const { t } = useTranslation();
+  const [clusterName, setClusterName] = React.useState<string>('');
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/views/Targets/BrowseCluster/TableData.tsx:21]: This state variable is not needed

File: litmus-portal/frontend/src/views/Targets/BrowseCluster/TableData.tsx [hunk 2]
```diff
@@ -30,6 +31,7 @@ const TableData: React.FC<TableDataProps> = ({ data, deleteRow }) => {
   const [open, setOpen] = React.useState(false);
 
   const handleClick = () => {
+    setClusterName(data.cluster_name);
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/views/Targets/BrowseCluster/TableData.tsx:34]: This line is not needed

File: litmus-portal/frontend/src/views/Targets/BrowseCluster/TableData.tsx [hunk 3]
```diff
@@ -121,15 +123,22 @@ const TableData: React.FC<TableDataProps> = ({ data, deleteRow }) => {
...
                 <div className={classes.body}>
-                  <img src="/icons/bin-red-delete.svg" alt="Delete" />
+                  <img src="/icons/DisconnectIcon.svg" alt="disconnect" />
                   <div className={classes.text}>
                     <Typography className={classes.typo} align="center">
                       {t('targets.modalDelete.head1')} <br />
                       <strong> {t('targets.modalDelete.head2')}</strong>
+                      <strong>
+                        {' '}
+                        {clusterName} {t('targets.modalDelete.head4')}
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/views/Targets/BrowseCluster/TableData.tsx:133]:
```suggestion
                        {data.cluster_name} {t('targets.modalDelete.head4')}
```

File: litmus-portal/frontend/src/views/Targets/BrowseCluster/TableData.tsx [hunk 4]
```diff
@@ -121,15 +123,22 @@ const TableData: React.FC<TableDataProps> = ({ data, deleteRow }) => {
...
               >
                 <div className={classes.body}>
-                  <img src="/icons/bin-red-delete.svg" alt="Delete" />
+                  <img src="/icons/DisconnectIcon.svg" alt="disconnect" />
                   <div className={classes.text}>
                     <Typography className={classes.typo} align="center">
                       {t('targets.modalDelete.head1')} <br />
                       <strong> {t('targets.modalDelete.head2')}</strong>
+                      <strong>
+                        {' '}
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/views/Targets/BrowseCluster/TableData.tsx:132]: also please remove the empty space

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

**Comment by oumkale** [comment]
PTAL @arkajyotiMukherjee

**Review by ishangupta-ds — APPROVED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**